samples/java_springboot/21.corebot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/FlightBookingRecognizer.java

```diff
@@ -31,6 +31,8 @@
      * @param telemetryClient The BotTelemetryClient to use.
      */
     public FlightBookingRecognizer(Configuration configuration, BotTelemetryClient telemetryClient) {
+        LuisRecognizer recognizer1;
+        recognizer1 = null;
```

Comment:
If these are the changes that we should apply just to have it as `final`, I'd prefer to rollback this